src/main/java/com/olgaboiar/mint/RequestParser.java

```diff
@@ -46,6 +48,11 @@ public String parseContentLength(Map<String, String> requestHeaders) {
         return requestHeaders.get("Content-Length").split(":")[0];
     }
 
+    public String parseContentType(List<String> incomingRequest) throws IOException {
+        String path = parsePath(parseRequestLine(incomingRequest));
+        return Files.probeContentType(Paths.get(path));
```

Comment:
What's the content-type value is returned if there's no filetype extension on the request path?

Reply:
it was returning null; Content-Type is optional and is added only if actual content is served since https://github.com/olgaboiar/MintHTTPServer/pull/14/commits/f69814852a141c7b701e3e7aeb70d83b893e2756